arangod/VocBase/Methods/Indexes.cpp

```diff
@@ -145,18 +147,23 @@ arangodb::Result Indexes::getAll(LogicalCollection const* collection,
     tmp.close();
 
   } else {
-    SingleCollectionTransaction trx(transaction::StandaloneContext::Create(
-                                        collection->vocbase()),
-                                    *collection, AccessMode::Type::READ);
-
-    // we actually need this hint here, so that the collection is not
-    // loaded if it has status unloaded.
-    trx.addHint(transaction::Hints::Hint::NO_USAGE_LOCK);
+    std::shared_ptr<transaction::Methods> trx;
+    if (inputTrx) {
+      trx = std::shared_ptr<transaction::Methods>(inputTrx, [](transaction::Methods*) {
+      } /*NoDelete<transaction::Methods>()*/);
```

Comment:
What is the meaning of this comment?

Reply:
Ah, sorry, this was an old piece of code from a since-abandoned WIP.